docs: Implement style guide rules - active voice, misc #2 (#23273)
MystenLabs/sui: docs/content/concepts/sui-architecture/sui-storage.mdx

| State-sync archival | Cloud storage        | State-sync archival stores blockchain state data in cloud storage to support syncing of historical objects that may have been pruned.                                        | ~1.3TB                                 |

## Validators

Validators should enable pruning to limit database growth. Transaction-per-second (TPS) rates affect growth, but storing full transaction and object history is the main factor. Transaction type also influences data growth.  

![Storage](./images/storage.png)  

The image above shows Mainnet growth in 2024:  

- Over a 90-day period with ~18 TPS, disk usage increased from 203GB to 215GB (~130MB per day).  
- Over a two-week period with ~183 TPS, disk usage grew from 215GB to 271GB (~4GB per day).  

These results show storage growth increases relative to TPS rates.  

## Pruning full nodes

Pruning full nodes are similar in disk usage to validators because pruning settings are the same. There are two exceptions:  

- Full nodes keep a `consensus_db`, which accounts for about half of validator disk usage.  

> - If the node serves RPC queries, the `indexes/` directory in Mainnet consumes significant space. As of today, `indexes/` is 1.5TB and grows with TPS.  

Author: ```suggestion
- If the node serves RPC queries, the `indexes/` directory in Mainnet consumes significant space. Currently, `indexes/` is 1.5TB and grows with TPS.  
```
